docker/docker-compose-aspnetcore/efcore/oats.efcore-blogs.yaml

```
@@ -0,0 +1,12 @@
+include:
+  - ./oats-template.yaml
+input:
+  - path: /api/EFCore/ListBlogs
+expected:
+  traces:
+    - traceql: '{ name =~ "api/EFCore/ListBlogs" }'
+      spans:
+        - name: 'GET'
```

```suggestion
        - name: 'main'
```

This change is required so tests pass for me.